package.json

```diff
@@ -83,6 +83,7 @@
     "prop-types": "^15.6.2",
     "react": "^16.4.2",
     "react-autosuggest": "^9.4.2",
+    "react-bootstrap": "^2.5.0",
```

Comment:
Uninstall this you are not using it

Reply:
Done!

Reply:
removed log file through git rm --cached